bin/GToTree

```diff
@@ -242,6 +242,16 @@ if [ -f "$NCBI_acc_file" ]; then
     fi
 fi
 
+# checking no duplicates labels in label file
+if [ -f "$file_to_genome_id_map" ]; then
+    num_dupes=$(cut -f 2 -d$'\t' "$file_to_genome_id_map" | uniq -d | wc -l)
+    if [ ! $num_dupes == 0 ]; then
+        printf "\n${RED}      $file_to_genome_id_map has duplicate labels on the second column, check it out and provide unique labels only.${NC}\n"
```

Comment:
Then because of the mix of columns 2 and 3, I'd make this something like:

`printf "\n${RED}      $file_to_genome_id_map appears to have duplicate labels in there, check it out and provide unique labels only.${NC}\n"`